src/resolvers/projectVerificationFormResolver.ts

```diff
@@ -0,0 +1,61 @@
+import { Arg, Ctx, Mutation, registerEnumType, Resolver } from 'type-graphql';
+import { MyContext } from '../types/MyContext';
+import { errorMessages } from '../utils/errorMessages';
+import {
+  createProjectVerificationRequestValidator,
+  validateWithJoiSchema,
+} from '../utils/validators/graphqlQueryValidators';
+import { logger } from '../utils/logger';
+import { findProjectById } from '../repositories/projectRepository';
+import {
+  createProjectVerificationRequest,
+  getInProgressProjectVerificationRequest,
+} from '../repositories/projectVerificationRepository';
+import { ProjectVerificationForm } from '../entities/projectVerificationForm';
+
+@Resolver(of => ProjectVerificationForm)
+export class ProjectVerificationFormResolver {
+  @Mutation(returns => ProjectVerificationForm)
+  async createProjectVerificationForm(
+    @Arg('projectId') projectId: number,
+    @Ctx() { req: { user } }: MyContext,
+  ): Promise<ProjectVerificationForm> {
+    try {
+      const userId = user?.userId;
+      if (!userId) {
+        throw new Error(errorMessages.UN_AUTHORIZED);
+      }
+      validateWithJoiSchema(
+        {
+          projectId,
+        },
+        createProjectVerificationRequestValidator,
+      );
+      const project = await findProjectById(projectId);
+      if (!project) {
+        throw new Error(errorMessages.PROJECT_NOT_FOUND);
+      }
+      if (Number(project.admin) !== userId) {
+        throw new Error(errorMessages.YOU_ARE_NOT_THE_OWNER_OF_PROJECT);
+      }
+      if (project.verified) {
+        throw new Error(errorMessages.PROJECT_IS_ALREADY_VERIFIED);
+      }
+
+      const inProjectVerificationRequest =
+        await getInProgressProjectVerificationRequest(projectId);
+      if (inProjectVerificationRequest) {
+        throw new Error(
+          errorMessages.THERE_IS_AN_ONGOING_VERIFICATION_REQUEST_FOR_THIS_PROJECT,
+        );
+      }
+      return createProjectVerificationRequest({
+        projectId,
+        userId,
+      });
+    } catch (e) {
+      logger.error('createProjectVerificationRequest() error', e);
+      throw e;
+    }
+  }
+}
```

Comment:
@FatemeGhasemi 
Please write some test cases here for these failed scenarios:
- [ ] When project is already verified and we throw error
- [ ] When projectId is not valid and can't find the project
- [ ] When user is not authenticated
- [ ] When user is not authorized (is not project's owner)
- [ ] When user already have an inprogress project verification form ( you should write two test cases for both `draft` and `submitted` statuses
And anything that can write test for that ( please see the codes and find possible errors and write test for them